plugins/module_utils/network/sonic/argspec/bgp_neighbors/bgp_neighbors.py

```diff
@@ -148,6 +148,24 @@ def __init__(self, **kwargs):
                                             },
                                             'type': 'dict'
                                         },
+                                        'ip_afi': {
+                                            'options': {
+                                                'default_policy_name': {'type': 'str'},
```

Comment:
These default route policy options are also needed for BGP neighbors.

Reply:
Yes, these attributes will be added to the bgp_neighbors_af module. The PR for that module will be opened today.

Reply:
Thanks for explaining. (As mentioned above, the current resource module structuring is confusing, but we can proceed with it for now.)